### ISSUE TITLE
getContent() crashes when called on an Article

### ISSUE THREAD

**Issue by BobRay** [modxcms/Articles#141]
I'm not sure if Articles is being maintained at all, but I thought I'd post this in case others run into the same problem. 

After retrieving an Article in code, calling getContent() on it throws a PHP error:

> Fatal error: Uncaught Error: Call to undefined method modDocument_mysql::getArchivistCall()

The bug comes from Article class' getContent() method which, loads the related container and calls these methods:

```
$container->getArchivistCall();
$container->getLatestCommentsCall();
$container->getLatestPostsCall();
$container->getTagListerCall();
```

IMO, the getContent() method in the Articles class (which overrides the modResource method of the same name) should be renamed, since it performs a number of functions unrelated to getting the content of a resource.

This is an issue because any utility code that iterates through a collection of modResource objects and calls getContent() on each one will crash if it hits an Article. 

Code to reproduce issue:

```
$doc = $modx->getObject('modResource', array('class_key' => 'Article'));
if ($doc) {
    echo "OK";
}

$content = $doc->getContent();

```

**Comment by Mark-H** [comment]
If [this line](https://github.com/modxcms/Articles/blob/develop/core/components/articles/model/articles/article.class.php#L57) is changed to `if ($container instanceof ArticlesContainer) {`, that should fix it as well. For some reason it appears that the container for the Article resource you're loading is not of the expected type.

**Comment by BobRay** [comment]
That's definitely an easier fix.

**Comment by JoshuaLuckers** [comment]
Hi @BobRay. I can't reproduce this bug, but if the change @Mark-H has mentioned works fine for you, please let me know!

**Comment by JoshuaLuckers** [comment]
This issue is fixed (5557ac957b6ed154def11cf0ca530d1cbb569025). I will release a new version of Articles that includes this fix very soon!
